src/librustc/ty/query/job.rs

```diff
@@ -437,11 +437,8 @@ pub unsafe fn handle_deadlock() {
     let syntax_pos_globals = &*syntax_pos_globals;
     thread::spawn(move || {
         tls::GCX_PTR.set(gcx_ptr, || {
-            syntax_pos::GLOBALS.set(syntax_pos_globals, || {
```

Comment:
Where did the `libsyntax` globals go? :/

Reply:
Not sure -- AFAICT, pre-existing bug? I can try and fix it though -- I imagine just call `syntax::with_globals` here instead of `syntax_pos::GLOBALS`?

Reply:
We want both `syntax::GLOBALS.set` and `syntax_pos::GLOBALS.set` here. One of the previous `syntax_pos` was supposed to be `syntax`.